doc/src/concepts/proxies.md

```diff
@@ -0,0 +1,29 @@
+# Proxies
+
+`rustup` provides a number of wrappers for common Rust tools.
+These are called _proxies_ and represent commands which are
+provided by the various [components][].
+
+The list of proxies is currently static in `rustup` and is as follows:
+
+[components]: concepts/components.html
+
+- `rustc`: `rustc` is the compiler for the Rust programming language, provided by the project itself and come the from the `rustc` component.
+
+- `rustdoc`: `rustdoc` is a tool distributed in the `rustc` component which helps you to generate documentation for Rust projects.
+
+- `cargo`: cargo is the rust package manager which downloads your Rust package’s dependencies, compiles your packages, makes distributable packages, and uploads them to crates.io and the Rust community’s package registry and it comes from the `cargo` component.
```

Comment:
Here you refer to 'rust' and to 'Rust' in the same sentence.  It'd be better to capitalise Rust consistently.